.github/workflows/build_conda_pkg.yml

```diff
@@ -45,16 +48,16 @@ jobs:
           python .github/conda_config.py "$(python setup.py --version)"
           cd evalml-core-feedstock
           echo "Pre docker login"
-          echo ${{ secrets.DOCKERHUB_PASSWORD }} | docker login -u ${{ secrets.DOCKERHUB_USER }} --password-stdin
+          echo "${DOCKERHUB_PASSWORD}" | docker login -u "${DOCKERHUB_USER}" --password-stdin
```

Comment:
The ${} is shell syntax for referencing a variable
The quotes are shell scripting best-practice, just in case our username or password contain special chars (which is what was happening this morning!)